tests/conftest.py

```diff
@@ -226,70 +110,37 @@ def setup_user(browser: webdriver.Firefox | webdriver.Chrome):
     if email and password:
         page.login(email=email, password=password)
 
+    WebDriverWait(browser, 10).until(EC.url_to_be(base_url + "contactList"))
 
-@pytest.fixture(scope="function")
-def create_contact_info(
-    browser: webdriver.Firefox | webdriver.Chrome, setup_user
-):
-    """
-    Creates contact information using the Faker library.
-    """
-
-    logger.info("Create contact.")
-    link = base_url + "addContact"
-    page = AddNewContactPage(browser=browser, url=link)
-    page.open()
-
-    fake = Faker()
-    fake_contact_first_name = fake.first_name()
-    fake_contact_last_name = fake.last_name()
-    fake_contact_date_of_birth = (
-        fake.date_of_birth(minimum_age=6, maximum_age=110)
-    ).strftime("%Y-%m-%d")
-    fake_contact_email = fake.email()
-    fake_contact_phone = fake.basic_phone_number()
-    fake_contact_street_address_1 = fake.street_name()
-    fake_contact_city = fake.city()
-    fake_contact_state = fake.state()
-    fake_contact_postal_code = fake.postalcode()
-    fake_contact_country = fake.country()[:40]
-
-    return (
-        fake_contact_first_name,
-        fake_contact_last_name,
-        fake_contact_date_of_birth,
-        fake_contact_email,
-        fake_contact_phone,
-        fake_contact_street_address_1,
-        fake_contact_city,
-        fake_contact_state,
-        fake_contact_postal_code,
-        fake_contact_country,
-    )
+    return page
 
 
 @pytest.fixture(scope="function")
 def created_contact(
     browser: webdriver.Firefox | webdriver.Chrome,
     setup_user,
-    create_contact_info,
-    pytestconfig,
 ):
     """
     Creates a new contact using Selenium.
     """
 
+    _ = setup_user
+
+    contact_info = ContactsHelper.fake_contact()
+
     logger.info(
         "Creating contact with Contact first name: %s, last name: %s",
-        create_contact_info[0],
-        create_contact_info[1],
+        contact_info["firstName"],
+        contact_info["lastName"],
     )
 
     add_new_contact_link = base_url + "addContact"
     page = AddNewContactPage(browser=browser, url=add_new_contact_link)
     page.open()
 
-    page.add_new_contact(*create_contact_info)
+    WebDriverWait(browser, 10).until(EC.url_to_be(base_url + "addContact"))
```

Comment:
Why are you using pure `webdriver` here(in a contest file), although you have a page object?